migrations/versions/b34aa62af7fc_uuid_columns_for_project_profile_user_team.py

```diff
@@ -0,0 +1,121 @@
+    column('id', sa.Integer()),
+    column('uuid', UUIDType(binary=False)),
+    column('userid', sa.Unicode(22)),
+    )
+
+team = table('team',
+    column('id', sa.Integer()),
+    column('uuid', UUIDType(binary=False)),
+    column('userid', sa.Unicode(22)),
+    )
+
+
+def get_progressbar(label, maxval):
+    return ProgressBar(
+        maxval=maxval,
+        widgets=[
+            label, ': ',
+            progressbar.widgets.Percentage(), ' ',
+            progressbar.widgets.Bar(), ' ',
+            progressbar.widgets.ETA(), ' '
+        ])
+
+
+def upgrade():
+    conn = op.get_bind()
+
+    op.add_column('project', sa.Column('uuid', UUIDType(binary=False), nullable=True))
+    count = conn.scalar(
+        sa.select([sa.func.count('*')]).select_from(project))
+    progress = get_progressbar("Projects", count)
+    progress.start()
+    items = conn.execute(sa.select([project.c.id]))
+    for counter, item in enumerate(items):
+        conn.execute(sa.update(project).where(project.c.id == item.id).values(uuid=uuid4()))
+        progress.update(counter)
+    progress.finish()
+    op.alter_column('project', 'uuid', nullable=False)
+    op.create_unique_constraint('project_uuid_key', 'project', ['uuid'])
+
+    op.add_column('profile', sa.Column('uuid', UUIDType(binary=False), nullable=True))
+    count = conn.scalar(
+        sa.select([sa.func.count('*')]).select_from(profile))
+    progress = get_progressbar("Profiles", count)
+    progress.start()
+    items = conn.execute(sa.select([profile.c.id, profile.c.userid]))
+    for counter, item in enumerate(items):
+        conn.execute(sa.update(profile).where(profile.c.id == item.id).values(uuid=buid2uuid(item.userid)))
+        progress.update(counter)
+    progress.finish()
+    op.alter_column('profile', 'uuid', nullable=False)
+    op.create_unique_constraint('profile_uuid_key', 'profile', ['uuid'])
+
+    op.add_column('team', sa.Column('uuid', UUIDType(binary=False), nullable=True))
+    count = conn.scalar(
+        sa.select([sa.func.count('*')]).select_from(team))
+    progress = get_progressbar("Teams", count)
+    progress.start()
+    items = conn.execute(sa.select([team.c.id, team.c.userid]))
+    for counter, item in enumerate(items):
+        conn.execute(sa.update(team).where(team.c.id == item.id).values(uuid=buid2uuid(item.userid)))
+        progress.update(counter)
+    progress.finish()
+    op.alter_column('team', 'uuid', nullable=False)
+    op.create_unique_constraint('team_uuid_key', 'team', ['uuid'])
+
+    op.add_column('user', sa.Column('uuid', UUIDType(binary=False), nullable=True))
+    count = conn.scalar(
+        sa.select([sa.func.count('*')]).select_from(user))
+    progress = get_progressbar("Users", count)
+    progress.start()
+    items = conn.execute(sa.select([user.c.id, user.c.userid]))
+    for counter, item in enumerate(items):
+        conn.execute(sa.update(user).where(user.c.id == item.id).values(uuid=buid2uuid(item.userid)))
```

Comment:
Why `buid2uuid` for these 3 cases?

Reply:
`userid` columns use buid renderings of UUIDs, so we migrate existing values. This is required to ensure the values match what is in Lastuser.